common.mk

```diff
@@ -41,7 +41,7 @@ LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Version
 
 # Add supported OS-ARCHITECTURE combinations here
 ENVS ?= linux-amd64 windows-amd64 darwin-amd64
-STANDALONE_PLUGINS ?= login management-cluster package pinniped-auth secret
+STANDALONE_PLUGINS ?= login management-cluster package pinniped-auth secret feature
```

Comment:
As the `feature` plugin is associated with `featuregate` management-package and it that can only be used if `featuregate` management-package is installed, I think it makes this plugin a context-aware plugin.

Reply:
makes sense, updated